kanopi/starter-8
# Composer manifest

## Description
> As a developer, I want to have a human-parsable log of what's installed in my repo, so that I can debug conflicts if necessary.

https://github.com/joachim-n/composer-manifest

From its readme:

>  When Composer performs an install or an update, the output tells you what has been changed. But once that output is gone, it's hard to reconstitute it, as a diff of the changes to the composer.lock file is hard to read.
> 
> This plugin writes a YAML file listing each package on one line, so a diff or log can show you exactly what has been changed.

## Acceptance Criteria


## Affected URL
NA

## Related Tickets
[Add composer manifest to starter](https://kanopi.teamwork.com/#tasks/22666474)


## Steps to Validate
1. Observe the new `composer-manifest.yml`
1. Try requiring or updating a package
1. Observe nice changes that you can see with a glance instead of grepping through 1000s of lines of `composer.lock`.

## Deploy Notes
NA
